natives.json

```diff
@@ -94694,33 +94694,33 @@
 			"build": "323"
 		},
 		"0x45C597097DD7CB81": {
-			"name": "ADD_COVER_BLOCKING_AREA",
+			"name": "ADD_COVER_BLOCKING_IN_ANGLED_AREA",

```

Comment:
I didn't notice it was confirmed. I reverted the name changes and I double checked the parameters are still correct. These natives must be referring to normal areas, not an angled areas as it seemed based on plotting positions the scripts were using in Codewalker.

Reply:
Also, I just realized these obviously cannot be angled areas because angled areas require an extra width float, which neither of these functions have.

Reply:
Yeah I read Alexander Blade's post on GTA Forums regarding native naming a while ago but forgot about that part. Good to see so many names have been confirmed already.